README.rst

```diff
@@ -53,21 +53,14 @@ Installation on Open edX Devstack
 
 	cd ~/Documents/eoxstack/devstack  # Change for your devstack path (if you are using devstack)
 	make lms-shell  # Enter the devstack machine (or server where lms process lives)
-	sudo su edxapp -s /bin/bash  # if you are using devstack you need to use edxapp user
-	source /edx/app/edxapp/venvs/edxapp/bin/activate
 	cd /edx/src/edxapp/eox-core
 	pip install -e .
 
 Development
```

Comment:
I think that we can change this title with something more related to the information that is below.

Reply:
Something like ''Remove unnecessary installation steps and update dependency management instructions." would do?